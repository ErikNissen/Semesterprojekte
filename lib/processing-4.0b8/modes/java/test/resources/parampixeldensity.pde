version https://git-lfs.github.com/spec/v1
oid sha256:dab22d138d16cfda04ba68db8a55de85e76e13fa3f31aecf8e6472ae1a9f399c
size 218
