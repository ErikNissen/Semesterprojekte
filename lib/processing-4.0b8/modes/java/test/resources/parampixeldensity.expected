version https://git-lfs.github.com/spec/v1
oid sha256:6007255b30773c2ecc049a5165225fd3430a2aca76e7c5be7fb3470fa358656a
size 1033
